<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="MainFrame">
      <property name="styleSheet">
       <string notr="true">background-color: #2b3e47;
border-radius: 25px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="classifyButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>490</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura Md BT</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 15pt &quot;Futura Md BT&quot;;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 17pt &quot;Futura Md BT&quot;;

}
</string>
       </property>
       <property name="text">
        <string>Classify</string>
       </property>
      </widget>
      <widget class="QFrame" name="circle1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>430</y>
         <width>351</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Images/circle1.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="logo">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>261</width>
         <height>141</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Images/logoAppWhite.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="circle2">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>60</y>
         <width>631</width>
         <height>661</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Images/circle1.png);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="robot">
        <property name="geometry">
         <rect>
          <x>-80</x>
          <y>10</y>
          <width>341</width>
          <height>291</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/Images/Images/robot.png);
background-color:transparent;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="circle3">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>190</y>
         <width>211</width>
         <height>171</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QFrame" name="circle4">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>360</y>
         <width>101</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="generateButton">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>490</y>
         <width>221</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Futura Md BT</family>
         <pointsize>15</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 15pt &quot;Futura Md BT&quot;;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font-size : 25px;
	font: 17pt &quot;Futura Md BT&quot;;
}
</string>
       </property>
       <property name="text">
        <string>Generate</string>
       </property>
      </widget>
      <widget class="QFrame" name="windowNav">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>10</y>
         <width>100</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="minimizeButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="home_screen.qrc">
            <normaloff>:/Images/Images/minimize.png</normaloff>:/Images/Images/minimize.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="home_screen.qrc">
            <normaloff>:/Images/Images/close.png</normaloff>:/Images/Images/close.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="main_header">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1111</width>
         <height>131</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
      <zorder>logo</zorder>
      <zorder>circle1</zorder>
      <zorder>circle2</zorder>
      <zorder>circle3</zorder>
      <zorder>classifyButton</zorder>
      <zorder>circle4</zorder>
      <zorder>generateButton</zorder>
      <zorder>main_header</zorder>
      <zorder>windowNav</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="home_screen.qrc"/>
 </resources>
 <connections/>
</ui>
